docs/topics/multiplatform/native-cocoapods.md

```diff
@@ -314,10 +314,75 @@ manually or using a shell command:
     echo -e "kotlin.apple.cocoapods.bin=$(which pod)" >> local.properties
     ```
 
-### Module not found {initial-collapse-state="collapsed" collapsible="true"}
+### Module or framework not found {initial-collapse-state="collapsed" collapsible="true"}
 
-You may encounter a `module 'SomeSDK' not found` error that is connected with the [C-interop](native-c-interop.md) issue.
-Try these workarounds to avoid this error:
+When installing Pods, you may encounter `module 'SomeSDK' not found` or `framework 'SomeFramework' not found`
+errors connected to [C interop](native-c-interop.md) issues. To resolve such errors, try these solutions:
+
+#### Update packages
+
+Update your installation tool and the installed packages (gems):
+
+<tabs>
+<tab title="RVM">
+
+1. Update Ruby version manager (RVM):
```

Comment:
The sequence seems out of order: we first use "RVM" as the title of the tab, and then explain the acronym inside it. Either the audience knows what RVM is, and doesn't need the explanation, or the title of the tab should be "Ruby version manager", although that is a bit long.

Also, to be extra nitpicky, Ruby Version Manager is capitalized all the way on the official website (in one or two places it is actually written out). Also, apparently the default name is "Ruby enVironment Manager", but let's not do that :)

Reply:
I agree that the sequence is not ideal, but I thought the full name is too long for the tab title
Anyway, this explanation is probably redundant, it's for the people who already use this tool, removed